### DISCUSSION TITLE
Learn PC Explore in 5 minutes or less

### DISCUSSION THREAD

**Discussion by brunosan** [microsoft/PlanetaryComputer#75] [Show and tell]
We've been getting request to do a quick rundown using the Explore app. I've just recorded this one. Please share your feedback of what else you'd love to see. And if you record a better one, we'll replace it with yours.

https://user-images.githubusercontent.com/434029/174443566-562fef02-4c92-4907-8194-3dc63806054c.mp4

**Comment by gilbertocamara** [comment]
Hi Bruno,

This is a nice introduction to Explore. It nicely shows  the capabilities of the tool. 

The idea of showing the python code to retrieve the asset is also nice. Would you consider having an R version as well?

Best regards
Gilberto 
============================
Prof Dr Gilberto Camara
Senior Researcher
National Institute for Space Research (INPE), Brazil
https://gilbertocamara.org/ <https://gilbertocamara.org/>
=============================

**Reply by TomAugspurger** [reply]
> The idea of showing the python code to retrieve the asset is also nice. Would you consider having an R version as well?

cc @mmcfarland. This would require a bit of design work, but I imagine it's doable.

One note: we found it a bit challenging or impossible to construct "regular" (py)stac queries from the state of the Explorer Window. So the explorer generates a cql2-json filter (at https://github.com/microsoft/PlanetaryComputerDataCatalog/blob/17c12ee9b539977196eb535a090a655a3931c1e0/src/pages/Explore/components/ExploreInHub/pythonSnippet.ts#L45) and passes it off to pystac-client, which is able to turn it into an ItemCollection / iterator of items.

So I think to implement this, rstac would need to accept cql2-json filters. xref https://github.com/radiantearth/stac-api-spec/blob/62003db924fe6630545400a41b62fe052d65b8ff/fragments/filter/README.md and http://www.opengis.net/spec/cql2/1.0/conf/cql2-json (the cql2-json link is erroring right now).
